### ISSUE TITLE
Textual transcript: Improve navigation via pager for <pb>s that are missing

### ISSUE THREAD

**Issue by gerritbruening** [faustedition/faust-web#74]
In the Facsimile / docTranscript you can browse from page to page.

The two text views do not allow for this when it comes to &lt;div>s. Examples:
- http://dev.faustedition.net/documentViewer.php?faustUri=faust://xml/document/faust/2/gsa_391098.xml&page=5&view=document_text
- http://dev.faustedition.net/documentViewer.php?faustUri=faust://xml/document/faust/2/gsa_391098.xml&page=11&view=document_text
- http://dev.faustedition.net/documentViewer.php?faustUri=faust://xml/document/faust/2/gsa_391098.xml&page=28&view=document_text

:pencil2: switched URLs to dev

**Comment by thvitt** [comment]
Currently, when navigating to page _n_, the web app looks for the file corresponding to the `<div>` containing the `<pb/>` that corresponds to the page. It then loads the file and jumps to the `<pb/>` (or rather its HTML equivalent). 

Your examples have two separate issues: 

1. For pages 5 and 11, the `<pb/>`s are in a higher-level `<div>`, and thus the file that is loaded is the one corresponding to _Dritter Act_. If you don't like this, we would need to get rid of these intermediate files completely by moving the content before/after the scene-level `<div>`s to the HTML files corresponding to the following/last scene. This would involve some quite risky re-ordering in faustedition/faust-gen-html which I don't like to do for the beta.
2. For the (empty) page 28, there is no corresponding `<pb/>` in the textual transcript, so we don't have a jump target. This can probably be implemented smoother once I've resolved #15.

In any case, just browsing to either the following or preceding page (4, 6, 10, 12, 27, 29) will jump to the correct position in the text.

**Comment by gerritbruening** [comment]
I see both points, and agree to postpone.

**Comment by thvitt** [comment]
1. pp. 5 and 11 are fixed by the new splitting
2. pp. 6 and 28 are empty and thus don't have a jump target, depends on #15

**Comment by thvitt** [comment]
With the fix to #15 the textual view's content simply will not change when browsing to a page that isn't referenced in the textual transcript.

**Comment by gerritbruening** [comment]
Seems to work well now, thanks!
